packages/core-mobile/app/new/features/swap/utils/evm/getSwapRateStream.ts

```diff
@@ -0,0 +1,141 @@
+import { Network } from '@avalabs/core-chains-sdk'
+import { Account } from 'store/account'
+import { fetch as expoFetch } from 'expo/fetch';
```

Comment:
I assume you had to use expo fetch to handle stream response. if so, could you add a comment in the code?